infra/src/feature-env.ts

```diff
@@ -1,5 +1,7 @@
 import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
```

Comment:
What's that all about?

Reply:
just changed it from using require into import and moved it next to the yargs import